include/field.hxx

```diff
@@ -359,6 +359,46 @@ inline BoutReal min(const T& f, bool allpe, REGION rgn) {
   return min(f, allpe, toString(rgn));
 }
 
+
+template<typename T, typename = bout::utils::EnableIfField<T>>
+inline bool isConst(const T& f, bool allpe = false, const std::string& region = "RGN_ALL") {
```

Comment:
I disagree, I'm afraid! In my experience `const` is only ever pronounced "const" in the C++ community, never "constant", so my view is that they aren't ambiguous, but you may be right that they aren't sufficiently different. At any rate, I think I'm going to have to veto `const` in the name, way too similar to `std::is_const`/`std::as_const` for my liking

I'd prefer something slightly clumsy over ambiguous. Naming things is hard!

Some suggestions:
- `isScalar`/`getAsScalar`
- `isUnvarying`/`getAsUnvarying`
- `hasNoVariation`/`getNonvarying`
- `isUniform`/`getUniform`
- `isHomogeneous`/`getHomogeneous`

Very happy to hear other alternatives

Reply:
I was tempted by `isScalar` - but all the fields are scalar fields -.-
I think I would slightly prefer `isUniform` over `isHomogenous` because it is easier to spell, but both are my favourites.